include/morphio/dendritic_spine.h

```diff
@@ -0,0 +1,32 @@
+#pragma once
+
+#include <string>
+#include <vector>
+
+#include <morphio/morphology.h>
+#include <morphio/types.h>
+
+namespace morphio {
+
+class DendriticSpine: public Morphology
+{
+  public:
+    DendriticSpine(const std::string& source);
+
+    const std::vector<Property::DendriticSpine::PostSynapticDensity>& postSynapticDensity() const
+        noexcept;
+
+  protected:
+    friend class mut::DendriticSpine;
+    DendriticSpine(const Property::Properties& properties);
+
+  private:
+    Soma soma() const;
+    Mitochondria mitochondria() const;
```

Comment:
From my understanding, there is no Mitochondria and no Endoplasmic Reticulum object attached to Dendritic Spine.

Reply:
Yeah, that's why they are marked private so if anyone tries to access them, they won't be able to.